README.md

```diff
@@ -284,44 +351,49 @@ The **pointfinder.tsv** output file generated by `staramr` contains the followin
 | SRR1952908  | gyrA (S83Y)  | ciprofloxacin I/R, nalidixic acid  | codon  | 83        | TCC -> TAC (S -> Y)  | 99.96      | 100.00    | 2637/2637                | contig00008  | 22801   | 20165  |
 | SRR1952926  | gyrA (S83Y)  | ciprofloxacin I/R, nalidixic acid  | codon  | 83        | TCC -> TAC (S -> Y)  | 99.96      | 100.00    | 2637/2637                | contig00011  | 157768  | 160404 |
 
+## plasmidfinder.tsv
+
+The **plasmidfinder.tsv** output file generated by `staramr` contains the following columns:
+
+* __Isolate ID__: The id of the isolate/genome file(s) passed to `staramr`.
+* __Gene__: The particular AMR plasmid gene detected.
```

Comment:
Remove **AMR** here in in below lines for plasmidfinder section.

Reply:
Solved in commit 2202e45